docs/conf/portland/2022/team.rst

```diff
@@ -7,74 +7,6 @@ Meet the Team
 Our conference is run by a dedicated team consisting mostly of volunteers.
 This page outlines the folks who are helping get things done, and what their roles are.
 
-Folks
------
-
-Eric Holscher
-~~~~~~~~~~~~~
-
-{% include "bios/eric.rst" %}
-
-Samuel Wright
-~~~~~~~~~~~~~
-
-{% include "bios/sam.rst" %}
-
-Mo Nishiyama
-~~~~~~~~~~~~
-
-{% include "bios/mo.rst" %}
-
-Alicia Duncan
-~~~~~~~~~~~~~
-
-{% include "bios/alicia.rst" %}
-
-Christy Lutz
-~~~~~~~~~~~~
-
-{% include "bios/christy.rst" %}
-
-Rose Williams
-~~~~~~~~~~~~~
-
-{% include "bios/rose.rst" %}
-
-Jennifer Rondeau
-~~~~~~~~~~~~~~~~
-
-{% include "bios/jennifer.rst" %}
-
-Sasha Romijn
-~~~~~~~~~~~~
-
-{% include "bios/sasha.rst" %}
-
-Mikey Ariel
-~~~~~~~~~~~~~
-
-{% include "bios/mikey.rst" %}
-
-Bio coming soon
-
-Primary Roles
--------------
-
-* **Conference chair** - Eric Holscher
-* **Conference co-chair** - *TBD*
-* **Code of conduct response** - See `code of conduct </code-of-conduct/#reporting-and-contact-information>`_ page
-* **Speaker coordinator** - Samuel Wright
-* **Unconference coordinator** - TBD
-* **Lightning Talk coordinator** - TBD
-* **Infrastructure coordinator** - TBD
-* **Swag coordinator** - TBD
-* **Communication coordinator** - TBD
-* **Job Fair coordinator** - TBD
-* **Welcome Wagon coordinators** - TBD
-* **Sponsorship coordinator** - TBD
-* **Writing Day coordinator** - TBD
-* **Volunteer coordinator** - TBD
-* **Emcee** - TBD
-* **Social media coordinator** - TBD
+{% include "conf/portland/team.rst" %}
```

Comment:
I tried extracting this information to an included page but the preview build isn't pulling it for some reason :(

Reply:
Seems it changes each year? I don't think this works well for an include. 

It looks like the include file is empty though, which is likely the issue.